src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
```

Comment:
those variables can be more clear
Would be better to make it like smallDiscount/bigDiscount to make variable names more understandable